pkg/monitor/dynkeepalived.go

```diff
@@ -9,22 +9,29 @@ import (
 	"syscall"
 	"time"
 
+	"gopkg.in/yaml.v2"
+
 	"github.com/google/go-cmp/cmp"
 	"github.com/openshift/baremetal-runtimecfg/pkg/config"
 	"github.com/openshift/baremetal-runtimecfg/pkg/render"
 	"github.com/sirupsen/logrus"
 )
 
-const keepalivedControlSock = "/var/run/keepalived/keepalived.sock"
-const cfgKeepalivedChangeThreshold uint8 = 3
-const dummyPortNum uint16 = 123
-const unicastPatternInCfgFile = "unicast_peer"
+const (
+	keepalivedControlSock               = "/var/run/keepalived/keepalived.sock"
+	cfgKeepalivedChangeThreshold uint8  = 3
+	dummyPortNum                 uint16 = 123
+	unicastPatternInCfgFile             = "unicast_peer"
+	modeUpdateFilepath                  = "/etc/keepalived/monitor.conf"
+	userModeUpdateFilepath              = "/etc/keepalived/monitor.conf.d"
```

Comment:
I don't think this is a good filename. '.d' is usually meant to be a directory that you drop files into. What is the rationale for using this particular name?